### DISCUSSION TITLE
Icon使用iconfont.cn会不显示 求解蟹蟹 ≧( ° ° )≦

### DISCUSSION THREAD

**Discussion by hcisme** [ant-design/ant-design#35087] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/101104339/163939774-09f5625a-035f-498b-b38a-377d1e85064f.png)
![image](https://user-images.githubusercontent.com/101104339/163939826-4569428f-80c7-42d0-8e5c-eb7bb9184a09.png)
![image](https://user-images.githubusercontent.com/101104339/163939903-c14aa8b0-adc4-43ed-977b-7a4684229718.png)
![image](https://user-images.githubusercontent.com/101104339/163940341-25327ccf-fdcd-40ce-b754-e7eb5a941d07.png)
当我引入 iconfont.cn 的图标时 <IconFont type="icon-bilibili" style={{ fontSize: '1.2rem', padding: '.9rem' }} /> 并不会生效
求解答 蟹蟹 ≧( ° ° )≦

**Comment by embryoJuliy** [comment, marked as answer]
你需要引入的是js，而不是css

**Reply by hcisme** [reply]
好的感谢

**Comment by yoyo837** [comment]
```diff
- .css
+ .js
```

**Reply by hcisme** [reply]
好的感谢
